mev_inspect/inspect_block.py

```diff
@@ -75,6 +76,10 @@ def inspect_block(
         delete_arbitrages_for_block(db_session, block_number)
         write_arbitrages(db_session, arbitrages)
 
+    liquidations = get_liquidations(classified_traces)
```

Comment:
you're importing "get_liquidations" from "aave_liquidations," which I could see creating confusion in the future

is this for all liquidations? Or just aave liquidations?

if it is for aave liquidations I would rename to something like get_aave_liquidations